compiler/src/test/scala/org/scalajs/core/compiler/test/JSExportTest.scala

```diff
@@ -11,6 +11,8 @@ class JSExportTest extends DirectTest with TestHelpers {
 
   @Test
   def noDoubleUnderscoreExport = {
+    // scalastyle:off
```

Comment:
Hmmm... if it spills to the next files, there are some of the Big Numerics test which also need their flag stuff changed..